src/fonts/OpenSansSB_24px.h

```diff
@@ -0,0 +1,199 @@
+// Created by http://oleddisplay.squix.ch/ Consider a donation
```

Comment:
Isn't possible to re-use some parts from previous fonts and make a common out of it?